_unittest/test_21_Circuit.py

```diff
@@ -98,6 +98,8 @@ def test_06a_create_setup(self):
         assert LNA_setup.name == "LNA"
 
     def test_06b_add_3dlayout_component(self):
+        setup = self.aedtapp.create_setup("LNA")
```

Comment:
maybe you should use a different setup name for each unit test. "LNA" I think is already used in previous tests.

Reply:
Done!